app/tasks.py

```diff
@@ -111,16 +134,21 @@ def parseDate(date_f):
         value_f             = unicode(row[27], "utf-8", errors='ignore')
         customer_ref_f      = unicode(row[28], "utf-8", errors='ignore')
 
-        donor_f = getCreateDonor(donor_name_f, email_f, want_receipt_f, telephone_number_f,
-                                    mobile_number_f, address_line_f, city_f, province_f, postal_code_f, customer_ref_f)
+        telephone_number_f, telephone_extension_f = getTeleExt(telephone)
```

Comment:
:+1:

Reply:
got you! gonna fix it at the meeting tmr!